packages/go/analysis/ad/esc6.go

```diff
@@ -157,7 +157,10 @@ func PostCanAbuseUPNCertMapping(operation analysis.StatTrackedOperation[analysis
 					} else {
 						for _, dcForNode := range dcForNodes {
 							if cmmrProperty, err := dcForNode.Properties.Get(ad.CertificateMappingMethodsRaw.String()).Int(); err != nil {
-								log.Warnf("error in PostCanAbuseUPNCertMapping: unable to fetch %v property for node ID %v: %v", ad.StrongCertificateBindingEnforcementRaw.String(), dcForNode.ID, err)
+								log.Warnf("error in PostCanAbuseUPNCertMapping: unable to fetch %v property for node ID %v: %v", ad.CertificateMappingMethodsRaw.String(), dcForNode.ID, err)
+								continue
+							} else if cmmrProperty == -1 {
```

Comment:
Lot of magic numbers in here, some explained in comment and some not. Would you mind spending an additional pass on pulling these out into documented and named constants?

Reply:
I thought about that for the `-1` but it wasn't a constant else where and didn't know if it was intended as such. On it! 👍